keyboards/handwired/naho6key/keymaps/default/keymap.c

```diff
@@ -0,0 +1,117 @@
+#include "../../naho6key.h"
+#include "../../config.h"
+// deprecated includes when I sent actual strings instead of macro kbd
+// #include "keymap_french.h"
+// #include "sendstring_french.h"
+
+// layers
+#define _BASE 0
+#define _EMOJI 1
+#define _NAV 2
+#define _MOUSE 3
+#define _MEDIA 4
```

Comment:
We highly recommend enums instead of defines as they're easier to manage, long term. 

```suggestion
enum custom_layers {
    _BASE,
    _EMOJI,
    _NAV,
    _MOUSE,
    _MEDIA,
};
```